docs/dev/contributed-modules.md

```diff
@@ -0,0 +1,118 @@
+- Stable releases should be covered by the [security advisory policy](https://www.drupal.org/drupal-security-team/security-advisory-process-and-permissions-policy)
+- Choose modules with more than one maintainer. 
+- Look at number of commits by maintainers and historical activity. There is no specific limit, but if there is a low number of commits it might be something to flag.
+- Look at download numbers and number of sites using the module for an indication of usage. 
+- Look at the time since the last commit. We are looking for commits made in the last month. If there have been no new commits in the last three months, consider not using the module unless functionality is very simple and the module is considered "feature complete".
+- Ensure there is a non-beta non-dev release available i.e. a stable release.
+- Look at open issues, closed issues, and open bugs to ensure functionality we are seeking from the module is not impacted by any of these issues. Frequency of opened and closed issues is also an indicator of how well maintained the module is.
+- Make a "decision record" in our [contributed module decision records](#contributed-module-decision-records) section below. 
+
+## Contributed module decision records
+
+Here is where we keep documentation on modules we have added and descriptions of what we use them for. This does not include drupal core modules, composer, or drush. 
+
+### [`auto_entitylabel`](https://www.drupal.org/project/auto_entitylabel)
+
+_Added in April 2024_
+
+This module allows us to hide entity field labels. It was first added so that we could have an empty title field label for the WFO promo. We want to categorize WFO promo by WFO taxonomy and not necessarily have a title label. We realized this would be a helpful module for any entities that had blank, hidden, or alternate labels.
+
+### [`autologout`](https://www.drupal.org/project/autologout)
+
+_Added in January 2024_
+
+This module allows us to be compliant with AC-02(5) "Inactivity Logout", which requires us to set a 90 minute timeout period on user sessions. This is the most used module providing this functionality. 
+
+### [`config_split`](https://www.drupal.org/project/config_split)
+
+_Added in January 2024_
+
+We were looking for a way to disable certain modules locally (namely, samlauth and autologout) which only pertain to our cloud environments. We also wanted to set different configurations for autologout timing in beta than other cloud environments, fine tuning our settings for our production environment. This also allows us to restrict certain modules to dev-use only like `devel`. We could also use the `--no-dev` flag, but since the composer install in cloud.gov is not something we directly control, this flag is not used. `config_split` is the primary module for splitting configuration across environments. It required a complete reorganization of our configuration files. 
```

Comment:
Oh. So cloud.gov installs it anyway. Well then, seems like config-split makes the most sense then. 😂